devito/interfaces.py

```diff
@@ -474,11 +474,17 @@ class MemmapManager():
     """Class for managing all memmap related settings"""
     # used to enable memmap as default
     _use_memmap = False
+    # determine whether created files are deleted by default
+    _delete_file = True
     # flag for registering exit func
     _registered = False
+    # default directory to store memmap file
     _default_disk_path = os.path.join(gettempdir(), "devito_disk")
     # contains str name of all memmap file created
-    _created_data = set()
+    _created_data = {}
+    # unique id
+    _id = 0
+    # exit code used for normal exiting
```

Comment:
This class is getting increasingly complex, so I would propose to move it to it's own file.